examples/08-NBEATS-examples.ipynb

```diff
@@ -1,5 +1,15 @@
 {
```

Comment:
<p>The intro notebook has been renamed.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/unit8co/darts/pull/270/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Perfect, changed.